src/plots/cartesian/axes.js

```diff
@@ -944,9 +952,61 @@ axes.calcTicks = function calcTicks(ax, opts) {
             axes.prepTicks(mockAx, opts);
         }
 
+        if(mockAx.tickmode === 'full domain') {
+            var nt = mockAx.nticks;  // does mockAx have nitkcs?
+            if(nt === undefined) nt = 0;
+            if(nt === 0) {
+              // pass
```

Comment:
For `tickmode: 'auto'`, `nticks: 0` or `undefined` is "super automatic":

https://github.com/plotly/plotly.js/blob/620a56bcc043e3391e6649bbab1b8597a8ca07e4/src/plots/cartesian/axes.js#L671-L683

ie ending up somewhere between 5 and 10 depending on the graph size. For `full domain` I wouldn't want us dividing the axis in 7ths though, so maybe just set it to 5 or something? If you really want no ticks just hide them 😉 

This also reminds me: the new tickmodes should not be allowed at all on category or multicategory axes.

Reply:
Silent fail or throw error if they tried to put tickmode in category or multicategory?